packages/run-it/src/RunIt.tsx

```diff
@@ -260,6 +268,18 @@ export const RunIt: FC<RunItProps> = ({
             <PerfTracker perf={perf} configurator={configurator} />
           </TabPanel>
         )}
+        {isExtension ? (
+          <></>
+        ) : (
+          <TabPanel key="config">
```

Comment:
As an aside, Lara Harrow was showing me Tabs2 component today. Much easier to use.

Reply:
Oh cool. I'll upgrade to that tomorrow.

Reply:
`Tabs2` isn't quite ready yet. I'll create a ticket for that.